*** Makefile	Fri Jun  8 16:07:27 2001
--- Makefile.linux	Fri Jun  8 16:07:27 2001
***************
*** 2,10 ****
  #
  # calc - arbitrary precision calculator
  #
! # (Generic calc makefile)
  #
! # Copyright (C) 1999  Landon Curt Noll
  #
  # Calc is open software; you can redistribute it and/or modify it under
  # the terms of the version 2.1 of the GNU Lesser General Public License
--- 2,14 ----
  #
  # calc - arbitrary precision calculator
  #
! # (Linux calc makefile)
  #
! #	NOTE: The Makefile.linux is the Makefile used to make the calc rpm.
! #	      It differs from the standard calc Makefile in the ''You may
! #	      want to change some values below'' section.
! #
! # Copyright (C) 2001  Landon Curt Noll
  #
  # Calc is open software; you can redistribute it and/or modify it under
  # the terms of the version 2.1 of the GNU Lesser General Public License
***************
*** 21,28 ****
  # 59 Temple Place, Suite 330, Boston, MA  02111-1307, USA.
  #
  MAKEFILE_REV= $$Revision: 29.36 $$
! # @(#) $Id: Makefile.ship,v 29.36 2001/06/08 23:00:19 chongo Exp $
! # @(#) $Source: /usr/local/src/cmd/calc/RCS/Makefile.ship,v $
  #
  # Under source code control:	1990/02/15 01:48:41
  # File existed as early as:	before 1990
--- 25,32 ----
  # 59 Temple Place, Suite 330, Boston, MA  02111-1307, USA.
  #
  MAKEFILE_REV= $$Revision: 29.36 $$
! # @(#) $Id: Makefile.linux,v 29.36 2001/06/08 23:00:19 chongo Exp $
! # @(#) $Source: /usr/local/src/cmd/calc/RCS/Makefile.linux,v $
  #
  # Under source code control:	1990/02/15 01:48:41
  # File existed as early as:	before 1990
***************
*** 669,676 ****
  #
  # If in doubt, set USE_READLINE, READLINE_LIB and READLINE_INCLUDE to nothing.
  #
! USE_READLINE=
! #USE_READLINE= -DUSE_READLINE
  #
  #READLINE_LIB=
  READLINE_LIB= -lreadline -lhistory -lncurses
--- 673,680 ----
  #
  # If in doubt, set USE_READLINE, READLINE_LIB and READLINE_INCLUDE to nothing.
  #
! #USE_READLINE=
! USE_READLINE= -DUSE_READLINE
  #
  #READLINE_LIB=
  READLINE_LIB= -lreadline -lhistory -lncurses
***************
*** 683,692 ****
  
  # If $PAGER is not set, use this program to display a help file
  #
! CALCPAGER= more
  #CALCPAGER= pg
  #CALCPAGER= cat
! #CALCPAGER= less
  
  # Debug/Optimize options for ${CC} and ${LCC}
  #
--- 687,696 ----
  
  # If $PAGER is not set, use this program to display a help file
  #
! #CALCPAGER= more
  #CALCPAGER= pg
  #CALCPAGER= cat
! CALCPAGER= less
  
  # Debug/Optimize options for ${CC} and ${LCC}
  #
***************
*** 702,714 ****
  #
  #DEBUG= -O2
  #DEBUG= -O2 -g
! DEBUG= -O2 -g3
  #DEBUG= -O2 -ipa
  #DEBUG= -O2 -g3 -ipa
  #
  #DEBUG= -O3
  #DEBUG= -O3 -g
! #DEBUG= -O3 -g3
  #DEBUG= -O3 -ipa
  #DEBUG= -O3 -g3 -ipa
  #
--- 706,718 ----
  #
  #DEBUG= -O2
  #DEBUG= -O2 -g
! #DEBUG= -O2 -g3
  #DEBUG= -O2 -ipa
  #DEBUG= -O2 -g3 -ipa
  #
  #DEBUG= -O3
  #DEBUG= -O3 -g
! DEBUG= -O3 -g3
  #DEBUG= -O3 -ipa
  #DEBUG= -O3 -g3 -ipa
  #
